File: 1221-split-a-string-in-balanced-strings/1221-split-a-string-in-balanced-strings.js
/**
 * @param {string} s
 * @return {number}
 */
var balancedStringSplit = function(s) {
    let count=0
    s=s.split("")
    for(let i=0;i<s.length;i++){
        let temp=s.slice(0,i+1)
        const l=temp.filter((letter)=>letter==='L')
        const r=temp.filter((letter)=>letter==='R')
        if (l.length === r.length)count++
    }
    return count
};

File: 1221-split-a-string-in-balanced-strings/1221-split-a-string-in-balanced-strings.py
class Solution:
    def balancedStringSplit(self, s: str) -> int:
        count=0
        i=0
        s=list(s)
        while i <len(s):
            temp=s[:i+1]
            r=temp.count('R')
            l=temp.count('L')
            
            count+=1 if r==l  else 0
            i+=1

        return count
        